www/source/docs/reference/reference.html.md.erb

```diff
@@ -1260,13 +1260,45 @@ Packages are installed in the `/hab/pkgs/` directory, and then further organized
 ---
 ## <a name="sup-log-configuration" id="sup-log-configuration" data-magellan-target="sup-log-configuration">Supervisor Log Configuration Reference</a>
 
-With the 0.83.0 release of the Chef Habitat Supervisor, it is possible to have greater control over logging output, including the ability to dynamically adjust the logging configuration of a running Supervisor. There are two main ways of configuring logging, each of which has their own strengths and weaknesses: using environment variables, and using a configuration file.
+With the 0.83.0 release of the Chef Habitat Supervisor, it is possible to have greater control over logging output, including the ability to dynamically adjust the logging configuration of a running Supervisor. There are three ways of configuring logging, each of which has their own strengths and weaknesses: as part of the `hab` command, by using environment variables, and by using a configuration file.
 
-### Environment Variable Configuration
+### RUST_LOG with a command
+
+Use `RUST_LOG=debug RUST_BACKTRACE=1` to see an individual command's debug and backtrace.
+
+    ```bash
+    # Linux/MacOS
+    # replace "hab sup run" with your command
+    RUST_LOG=debug RUST_BACKTRACE=1 hab sup run
+    ```
+
+### RUST_LOG as an Environment Variable Configuration
+
+Use `RUST_LOG=debug RUST_BACKTRACE=1` to see an individual command's debug and backtrace.
+
+    ```bash
+    # Linux/MacOS
+    # replace "hab sup run" with your command
+    RUST_LOG=debug RUST_BACKTRACE=1 hab sup run
+    ```
```

Comment:
This seems like the content above was inadvertently duplicated?

Reply:
Note that `$env:RUST_LOG="debug"; $env:RUST_BACKTRACE=1; hab sup run` will indeed works but has some very different behavior from `RUST_LOG=debug RUST_BACKTRACE=1 hab sup run` in bash. The bash command will set those variables scoped to the hab process and will not leak them into the current shell. The powershell line will set those vars in the current shell and then the `hab` command will inherit them. the `;` s are just allowing you to run multiple single line commands on the same line. I believe the bash equivilent is `&&`. 

As mentioned above. I am in favor of removing instructions on how to set the vars on a single line because I think that could add more confusion than clarity. I think the important thing to communicate here is that `RUST_LOG` is an environment variable and setting it to `debug` will get you debug output. It is valuable to have both bash and powershell examples but I'm in favor of letting the users leverage their own OS knowledge to set those variables with different scopes.